programs/openbook-v2/src/instructions/create_market.rs

```diff
@@ -28,6 +29,19 @@ pub fn create_market(
             && (maker_fee.is_sign_positive() || maker_fee.abs() <= taker_fee),
         OpenBookError::InvalidFeesError
     );
+    require!(
+        time_expiry == 0 || time_expiry > Clock::get()?.unix_timestamp,
+        OpenBookError::MarketHasExpired
+    );
+    // Check if the decimals are greater than zero
+    require!(
+        ctx.accounts.base_mint.decimals > 0,
```

Comment:
the default market state for tests uses zero decimals

Reply:
what no, it uses 6 decimals with the `MintCookie`

Reply:
https://github.com/openbook-dex/openbook-v2/blob/52fac785f48d987d3db20ff07276395b61c03885/programs/openbook-v2/src/state/market.rs#L190